src/github/fetchBranches.js

```diff
@@ -35,9 +38,12 @@ module.exports = async ({ owner = github.owner, repository }) => {
 
   const {
     repository: {
+      defaultBranchRef: {
+        name
+      },
       refs: { nodes },
     },
   } = response.data.data;
 
-  return nodes;
+  return { nodes, defaultBranchName: name };
```

Comment:
It looks a bit confusing to see `defaultBranchName` when it is used together with `excludedBranches` in `filterStaleBranches.js`. Maybe, I should've named `excludedBranches` to `excludedBrancheNames`. 😆I'm fine either way of referring to the two but I wish their names looked consistent.

Reply:
haha sure let's keep it consistent. I'll rename `defaultBranchName` to `defaultBranch`